modules/builtin/src/content-types/_base.js

```diff
@@ -5,5 +5,19 @@ module.exports = {
       title: 'module.builtin.typingIndicator',
       default: true
     }
+  },
+  newRenderer: (data, type) => {
```

Comment:
I'll change the name to "renderer", but i'll wait before adding the deprecated tag because I have a bunch of changes in another PR related to channels, will do a final cleanup after those changes.